src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -831,7 +831,7 @@
     <value>Try this if there are issues with the shortcut (PowerToys Run might not get focus when triggered from an elevated window)</value>
   </data>
   <data name="PowerLauncher_ClearInputOnLaunch.Content" xml:space="preserve">
-    <value>Clear the previous query on launch</value>
+    <value>Clear the previous query on opening</value>
```

Comment:
I don't agree 'opening' is the correct verb here.  Invoke could be an alternative but PT Run is a surface that is temporal.

Reply:
Again, Microsoft guide. Talk to them if you think it's incorrect.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/exit